### DISCUSSION TITLE
GPIO Pin Mapping Inquiry for Raspberry Pi 3B (40-pin)

### DISCUSSION THREAD

**Discussion by RyzerZ** [Dan-in-CA/SIP#335] [Q&A]
I need to know, for each station, which GPIO pins to use for my Raspberry Pi 3B with 40 pins. I'm not understanding the GPIO map in the WIKI.

**Comment by Dan-in-CA** [comment]
Which plugin are you using to control the stations and what size relay board are you using?

**Comment by nnara530** [comment]
I am also trying to figureout which GPIO pins are used in the default SIP which is not using any plugin

**Reply by Dan-in-CA** [reply]
SIP without plugins uses GPO pins 4, 14, 17, 22, 27.
This is listed in the gpio_pns.py file in the gv.pin_map variable under rev 3, line 139
and is used for rain sensor and shift register control. see lines 188 and 208
